src/rules/ban_ts_ignore.rs

```diff
@@ -46,42 +46,31 @@ impl LintRule for BanTsIgnore {
 #[cfg(test)]
 mod tests {
   use super::*;
-  use crate::test_util::test_lint;
-  use serde_json::json;
+  use crate::test_util::*;
 
   #[test]
   fn ban_ts_ignore() {
-    test_lint(
-      "ban_ts_ignore",
+    assert_lint_err_on_line::<BanTsIgnore>(
       r#"
 // @ts-ignore
 function foo() {
   // pass
 }
-
+    "#,
+      "banTsIgnore",
```

Comment:
Passing expected code seems superfluous if we pass `BanTsIgnore` struct - this should probably be automated by adding `code()` method to `LintRule` trait; that way we can discover this code automatically. Fine for now, I'll see to it when refactoring `LintRule` trait.